datastream-server/src/main/java/com/linkedin/datastream/server/Coordinator.java

```diff
@@ -1001,6 +996,9 @@ private void handleLeaderDoAssignment() {
       instances.add(PAUSED_INSTANCE);
       _adapter.cleanupDeadInstanceAssignments(instances);
       _adapter.cleanupOldUnusedTasks(previousAssignmentByInstance, newAssignmentsByInstance);
+      if (cleanUpOrphanConnectorTasks) {
+        performCleanupTaskPostElection();
```

Comment:
can we rename this function now, since there is no longer a "CLEANUP_TASK_POST_ELECTION" event?

Reply:
Done.